### PR TITLE
Create a success page when after user log in

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@saeedmb8 | @miltontony Please review

File: molo/profiles/templates/registration/login.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h4 class="heading heading--subtitle">
 		<span class="input-hint">({% trans "Eg.: 2086" %})</span>
 	</fieldset>
 	<input type="submit" value="{% trans "Log in" %}">
-	<input type="hidden" name="next" value="{% if request.GET.next %}{{ request.GET.next }}{% else %}{{request.site.root_page.url}}{% endif %}" />
+	<input type="hidden" name="next" value="{% url "molo.profiles:login_success" %}" />
```
Comment (smarzban) [inline molo/profiles/templates/registration/login.html]: @Ntu2koTrevor we still need to keep the `{{ request.GET.next }}` if there is a next value

File: molo/profiles/templates/profiles/login_success.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h1 class="heading heading--profiles">{% trans "Login Success" %}</h1>
+<div class="profiles">
+    <h4 class="heading heading--feeback-title">{% trans "Login Successful!" %}</h4>
+    <a href="/">{% trans "Home Page" %}</a>
+  </div>
+{% endblock %}
```
Comment (smarzban) [inline molo/profiles/templates/profiles/login_success.html]: please add a new line

File: molo/profiles/tests/test_views.py [hunk 3]
```diff
@@ -90,6 +90,10 @@ def test_login(self):
         response = self.client.get(reverse('molo.profiles:auth_login'))
         self.assertContains(response, 'Forgotten your password')
 
+    def test_login_success(self):
```
Comment (smarzban) [inline molo/profiles/tests/test_views.py]: @Ntu2koTrevor you need to test here if a user logs in it will redirect them to the success page

**Comment by smarzban** [comment]
👍  🍰  🍬

**Pull request merged**